src/components/header/header.jsx

```diff
@@ -6,20 +6,25 @@ import Logo from './logo/logo'
 import Menu from './menu/menu'
 import MenuToggle from './menu-toggle/menu-toggle'
 import Search from './search/search'
+import style from './header.module.css'
 
-const Header = ({ menu, currentSection }) => {
+const Header = ({ menu, currentSection, background }) => {
   const [menuOpen, setMenuOpen] = useState(false)
   return (
     <>
-      <div className='w-full bg-gray-900 fixed z-20'>
+      <div
+        className={`${style.header} w-full fixed z-30 ${
+          background === 'solid' ? 'bg-gray-900' : ''
+        }`}
+      >
         <div className='max-w-screen-xl mx-auto px-6'>
-          <div className='-mx-6 h-16 flex items-center'>
+          <div className='-mx-6 h-20 flex items-center'>
```

Comment:
Need to make this configurable via a prop, since it's bigger on the landing page, and smaller on internal pages.